examples/discord_bot.py

```diff
@@ -19,8 +19,8 @@
 
 bot = commands.Bot(command_prefix="?", intents=discord.Intents.all())
 coc_client = coc.login(
-    os.environ["DEV_SITE_EMAIL"],
-    os.environ["DEV_SITE_PASSWORD"],
+    os.environ.get("DEV_SITE_EMAIL"),
```

Comment:
Please remove get because `os.environ` raises an exception if the environmental variable does not exist.whereas `os.environ.get` does not raise an exception, but returns None
Suggestion
```py
os.environ["DEV_SITE_EMAIL"]
```